open-machine-learning-jupyter-book/data-science/working-with-data/pandas2.ipynb

```diff
@@ -0,0 +1,2374 @@
+{
```

Comment:
<p>Is it making this part a public part? No need to keep the introduction section, I think so.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>